lib/algos/list/delimiter.js

```diff
@@ -133,6 +141,62 @@ class Delimiter {
     }
 }
 
+/**
+ * Extended delimiter class for versioning.
+ */
+class DelimiterVersions extends Delimiter {
+    /**
+     *  Constructor of the extension
+     *  Init and check parameters
+     *  @param {Object} parameters - parameters sent to DBD
+     *  @param {RequestLogger} logger - werelogs request logger
+     *  @param {object} latests - latest versions of some keys
```

Comment:
Can you call rename `latests` to `latestVersions` instead? It would be nice to be more descriptive and meaningful.